crates/db-boot/src/lib.rs

```diff
@@ -17,3 +17,14 @@ pub use x86_64::*;
 mod aarch64;
 #[cfg(target_arch = "aarch64")]
 pub use aarch64::*;
+
+/// Type for passing information about the initrd in the guest memory.
+pub struct InitrdConfig {
+    /// Load address of initrd in guest memory
+    pub address: vm_memory::GuestAddress,
+    /// Size of initrd in guest memory
+    pub size: usize,
+}
+
+/// Default (smallest) memory page size for the supported architectures.
+pub const PAGE_SIZE: usize = 4096;
```

Comment:
PAGE_SIZE should be defined per arch instead of a generic value. Some arch even have variable page size.

Reply:
Move it into x86_64/mod.rs?

Reply:
moved